app/src/main/java/com/rajatgoyal/puzzle15/ui/MainActivity.java

```diff
@@ -80,7 +81,14 @@ private void firebaseInit() {
     protected void onResume() {
         super.onResume();
 
-        if(isSignedIn()) return;
+        long prevGameTime = SharedPref.getGameTime();
```

Comment:
No need for this variable `prevGameTime` as it's not used more than once

Reply:
It gives more meaning to the value returned.